[2020-12-25 17:01:21,447] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: foodpanda.docker_op_tester 2020-12-25T09:01:17.107669+00:00 [queued]>
[2020-12-25 17:01:21,451] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: foodpanda.docker_op_tester 2020-12-25T09:01:17.107669+00:00 [queued]>
[2020-12-25 17:01:21,451] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2020-12-25 17:01:21,451] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2020-12-25 17:01:21,451] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2020-12-25 17:01:21,461] {taskinstance.py:1038} INFO - Executing <Task(DockerOperator): docker_op_tester> on 2020-12-25T09:01:17.107669+00:00
[2020-12-25 17:01:21,463] {standard_task_runner.py:51} INFO - Started process 29305 to run task
[2020-12-25 17:01:21,465] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'foodpanda', 'docker_op_tester', '2020-12-25T09:01:17.107669+00:00', '--job-id', '50', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ssh_practice.py', '--cfg-path', '/tmp/tmp06nydnov']
[2020-12-25 17:01:21,466] {standard_task_runner.py:76} INFO - Job 50: Subtask docker_op_tester
[2020-12-25 17:01:21,487] {logging_mixin.py:103} INFO - Running <TaskInstance: foodpanda.docker_op_tester 2020-12-25T09:01:17.107669+00:00 [running]> on host dataiku
[2020-12-25 17:01:21,503] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=arthur.sl.lin
AIRFLOW_CTX_DAG_ID=foodpanda
AIRFLOW_CTX_TASK_ID=docker_op_tester
AIRFLOW_CTX_EXECUTION_DATE=2020-12-25T09:01:17.107669+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2020-12-25T09:01:17.107669+00:00
[2020-12-25 17:01:21,513] {docker.py:224} INFO - Starting docker container from image hello-world
[2020-12-25 17:01:21,833] {docker.py:263} INFO - Hello from Docker!
This message shows that your installation appears to be working correctly.

To generate this message, Docker took the following steps:
 1. The Docker client contacted the Docker daemon.
 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
    (amd64)
 3. The Docker daemon created a new container from that image which runs the
    executable that produces the output you are currently reading.
 4. The Docker daemon streamed that output to the Docker client, which sent it
    to your terminal.

To try something more ambitious, you can run an Ubuntu container with:
 $ docker run -it ubuntu bash

Share images, automate workflows, and more with a free Docker ID:
 https://hub.docker.com/

For more examples and ideas, visit:
 https://docs.docker.com/get-started/
[2020-12-25 17:01:22,060] {xcom.py:238} ERROR - Could not serialize the XCom value into JSON. If you are using pickles instead of JSON for XCom, then you need to enable pickle support for XCom in your airflow config.
[2020-12-25 17:01:22,060] {taskinstance.py:1396} ERROR - Object of type 'bytes' is not JSON serializable
Traceback (most recent call last):
  File "/home/ub/airflow/env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1086, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/ub/airflow/env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1260, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/ub/airflow/env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1303, in _execute_task
    self.xcom_push(key=XCOM_RETURN_KEY, value=result)
  File "/home/ub/airflow/env/lib/python3.6/site-packages/airflow/utils/session.py", line 65, in wrapper
    return func(*args, session=session, **kwargs)
  File "/home/ub/airflow/env/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1833, in xcom_push
    session=session,
  File "/home/ub/airflow/env/lib/python3.6/site-packages/airflow/utils/session.py", line 62, in wrapper
    return func(*args, **kwargs)
  File "/home/ub/airflow/env/lib/python3.6/site-packages/airflow/models/xcom.py", line 88, in set
    value = XCom.serialize_value(value)
  File "/home/ub/airflow/env/lib/python3.6/site-packages/airflow/models/xcom.py", line 235, in serialize_value
    return json.dumps(value).encode('UTF-8')
  File "/usr/lib/python3.6/json/__init__.py", line 231, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib/python3.6/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib/python3.6/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib/python3.6/json/encoder.py", line 180, in default
    o.__class__.__name__)
TypeError: Object of type 'bytes' is not JSON serializable
[2020-12-25 17:01:22,061] {taskinstance.py:1440} INFO - Marking task as FAILED. dag_id=foodpanda, task_id=docker_op_tester, execution_date=20201225T090117, start_date=20201225T090121, end_date=20201225T090122
[2020-12-25 17:01:22,082] {local_task_job.py:118} INFO - Task exited with return code 1
